opentelemetry-api/src/opentelemetry/metrics/__init__.py

```diff
@@ -48,126 +52,104 @@ class Meter:
 
     def record_batch(
         self,
-        label_tuples: Dict[str, str],
-        record_tuples: List[Tuple["Metric", Union[float, int]]],
+        label_values: Tuple[str, ...],
+        record_tuples: Tuple[Tuple["Metric", ValueType]],
     ) -> None:
         """Atomically records a batch of `Metric` and value pairs.
 
         Allows the functionality of acting upon multiple metrics with
-        a single API call. Implementations should find handles that match
-        the key-value pairs in the label tuples.
+        a single API call. Implementations should find metric and handles that
+        match the key-value pairs in the label tuples.
 
         Args:
-            label_tuples: A collection of key value pairs that will be matched
-                against to record for the metric-handle that has those labels.
-            record_tuples: A list of pairs of `Metric` s and the
+            label_values: The values that will be matched against to record for
+                the handles under each metric that has those labels.
+            record_tuples: A tuple of pairs of `Metric` s and the
                 corresponding value to record for that metric.
         """
 
+    def create_metric(
+        self,
+        name: str,
+        description: str,
+        unit: str,
+        value_type: ValueType,
+        metric_kind: MetricKind,
+        label_keys: Tuple[str, ...] = None,
+        enabled: bool = True,
+        monotonic: bool = False,
```

Comment:
Weren't there different defaults for monotonic depending on the MetricKind?

Reply:
Yes. I pass the parameters from the child classes to the base class so they will get the corresponding default values.